package.json

```diff
@@ -1,38 +1,41 @@
 {
-  "name": "mithril-boilerplate",
+  "name": "mithril-breastplate",
   "description": "A quick setup for Mithril projects.",
-  "repository": "https://github.com/velveteer/mithril-boilerplate",
+  "repository": "https://github.com/soapdog/mithril-breastplate.git",
```

Comment:
Haha breastplate, that's clever. But you might want to change it back before I merge.